src/components/Share/Share.js

```diff
@@ -48,26 +54,36 @@ const renderConf = (conf, t) => (
 );
 
 const Share = ({ appBaseUrl }) => {
+  const activeTopic = useSelector(state => state.app.activeTopic);
+  const language = useSelector(state => state.app.language);
   const map = useSelector(state => state.app.map);
   const { t } = useTranslation();
-  const config = [...socialShareConfig];
+  const duplicateConfig = JSON.parse(JSON.stringify(socialShareConfig));
```

Comment:
Is that possible to have duplicate config ? why kind of person does that?

Reply:
without duplicating, we have initial language de.
We replace it in the url and have: 'url/de/...'
and if we change the language to 'it', replacing {lang} will not work anymore because it is already set ad 'de'.
and this won't update, because without duplicating we refer to the same object.

Reply:
So we always use the initial 'socialShareConfig' to replace, and our duplicate is not just a ref to the 'socialShareConfig'